core/app/models/spree/wishlist.rb

```diff
@@ -0,0 +1,43 @@
+module Spree
+  class Wishlist < Spree::Base
+    include SingleStoreResource
+
+    has_secure_token
+
+    belongs_to :user, class_name: Spree.user_class.to_s
+    belongs_to :store, class_name: 'Spree::Store', touch: true
```

Comment:
@MatthewKennedy do we need to touch the store? Probably it should be the other way around - we should expire user cache, not store in case they add/modify a Wishlist